charts/newrelic-infrastructure-beta/templates/kubelet/daemonset.yaml

```diff
@@ -0,0 +1,274 @@
+        {{- end }}
+        {{- if .Values.integrations_config }}
+        checksum/integrations_config: {{ include (print $.Template.BasePath "/kubelet/integrations-configmap.yaml") . | sha256sum }}
+        {{- end }}
+        {{- with .Values.podAnnotations }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
+      labels:
+        {{- include "newrelic.selectorLabels" . | nindent 8 }}
+        app.kubernetes.io/component: kubelet
+        {{- with .Values.podLabels }}
+          {{- toYaml . | nindent 8 }}
+        {{- end }}
+        mode: {{ include "newrelic.mode" . }}
+    spec:
+      {{- with .Values.images.pullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ template "newrelic.serviceAccountName" . }}
+      containers:
+        - name: kubelet
+          image: "{{ .Values.images.integration.registry }}/{{ .Values.images.integration.repository }}:{{ .Values.images.integration.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.images.integration.pullPolicy }}
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          env:
+            - name: "NRI_KUBERNETES_HTTPSERVERPORT"
+              value: "8003"
+            - name: "NRI_KUBERNETES_CLUSTERNAME"
+              value: {{ include "newrelic.cluster" . | required "cluster name must be set" }}
+            {{- if .Values.verboseLog }}
+            - name: NRI_KUBERNETES_VERBOSE
+              value: "1"
+            {{- end }}
+            - name: "NRI_KUBERNETES_NODENAME"
+              valueFrom:
+                fieldRef:
+                  apiVersion: "v1"
+                  fieldPath: "spec.nodeName"
+            # Required to connect to the kubelet
+            - name: "NRI_KUBERNETES_NODEIP"
+              valueFrom:
+                fieldRef:
+                  apiVersion: "v1"
+                  fieldPath: "status.hostIP"
+            {{- with .Values.kubelet.extraEnv }}
+            {{- toYaml . | nindent 12 }}
+            {{- end }}
+          {{- with .Values.kubelet.extraEnvFrom }}
+          envFrom: {{ toYaml . | nindent 12 }}
+          {{- end }}
+          volumeMounts:
+            - name: nri-kubernetes-config
+              mountPath: /etc/newrelic-infra/integrations.d/nri-kubernetes-config.yml
+              subPath: nri-kubernetes-config.yml
+            - mountPath: /tmp
+              name: tmpfs-tmp
+            {{- with .Values.kubelet.extraVolumeMounts }}
+            {{- toYaml . | nindent 12 }}
+            {{- end }}
+          {{- with .Values.kubelet.resources }}
+          resources: {{ toYaml . | nindent 12 }}
+          {{- end }}
+        - name: agent
+          image: "{{ .Values.images.agent.registry }}/{{ .Values.images.agent.repository }}:{{ .Values.images.agent.tag }}"
+          command: [ "/sbin/tini", "--", "/usr/bin/newrelic-infra" ]
```

Comment:
Does not this come from the image's entrypoint itself? I mean, if tomorrow (just a silly example) CAOS team changes the zombie reaper from `tini` to `surpervisord` we are having a failing chart because we are hardcoding it. I would remove it.
```suggestion
```